src/Containers/VendorsList/NavigationBar.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { StyledIcon } from '@styled-icons/styled-icon';
+import { NavigationItem, INavigationItemProps } from './NavigationItem';
+import { MainInterface, ResponsiveInterface } from '../../Utils/BaseStyles';
+import { flex, media, scroll } from '../../Utils/Mixins';
+
+
+export interface INavigationBarItems {
+    icon?: StyledIcon;
+    label: string
+};
+
+export interface INavigationBarProps extends MainInterface, ResponsiveInterface, React.HTMLAttributes<HTMLDivElement> {
+    navigationBarItems: INavigationBarItems[];
+    navigationItemProps?: INavigationItemProps;
+};
+
+const FIRST_LABEL = 0;
+
+export const NavigationBar: React.FC<INavigationBarProps> = ({
+    navigationBarItems,
+    navigationItemProps,
+    ...props
+}): React.ReactElement => {
+    const [selectedItem, setSelectedItem] = useState(navigationBarItems[FIRST_LABEL].label);
```

Comment:
Uncaught TypeError: Cannot read property 'label' of undefined at <anonymous>:1:7

Edge missing maybe? ?.label